Formula/knot-resolver.rb

```diff
@@ -26,15 +27,19 @@ class KnotResolver < Formula
   depends_on "hiredis" => :optional
   depends_on "libmemcached" => :optional
 
+  patch :DATA
```

Comment:
Could you use this patch, https://raw.githubusercontent.com/Homebrew/formula-patches/62168c0/knot-resolver/fix-loading-modules-on-Darwin.diff, instead of having the patch at the bottom of the formula, please. Also, it needs to be scoped to only be applied to stable builds since it is merged in head. The best way to do this is to wrap the `patch do` in an `if build.stable?` block.

Reply:
Done. For a future reference, could the formula-patches be added to https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md ?